spec/apps/invalid-mount/invalid-mount.app.js

```diff
@@ -15,6 +15,7 @@ export function mount() {
 export function unmount() {
   return new Promise(resolve => {
     unmountCalled = true;
+    resolve()
```

Comment:
This unmount wasn't getting called until my code change, but now is. So I had to fix a bug in it.